packages/myst-demo/src/index.tsx

```diff
@@ -110,17 +110,14 @@ async function parse(
     cite: { order: [], data: {} },
     footnotes: {},
   };
-  const { frontmatter: frontmatterRaw } = getFrontmatter(file, mdast, {
-    removeYaml: true,
-    removeHeading: options?.removeHeading ?? false,
```

Comment:
This is no longer an option in MyST, the heading is always removed if it matches `frontmatter.title`

For myst demo, this will cause funny behaviour in the specific case of matching `frontmatter.title` and first H1...

I think if we want to fix this, we should add the node back explicitly here in `myst-demo`.

Reply:
I think just testing it on any `{myst}` directives in the docs, I suppose that anything starting with an H1 is going to be a problem, if so, we will need to put it back in with any labels as well.

There are a few places we use frontmatter:
- https://github.com/executablebooks/mystjs/blob/main/docs/glossaries-and-terms.md#L53
- https://github.com/executablebooks/mystjs/blob/main/docs/math.md#L212
- https://github.com/executablebooks/mystjs/blob/main/docs/cross-references.md#L135

It might also be easier to just duplicate the logic of pulling the first yaml code block off?

Reply:
Ugh - ok, took your advice and duplicated the `getFrontmatter` logic directly in `myst-demo`. Might be worth adding more flexibility back to the other `getFrontmatter` version at some point...